lib/request.js

```diff
@@ -257,15 +257,14 @@ module.exports = {
    */
 
   get fresh() {
-    const method = this.method;
-    const s = this.ctx.status;
+    const { method, ctx: { status, response }, header } = this;
```

Comment:
Guys, `crazy` destruction is something like
```js
({ [{ method: { [dynoprope]: aVarName } }] } = anobject);
```
This `just a destruction`, developers that coming now to Javascript will see other way to write code as a bug. Don't be so conservative creating Koa for Node 7.6!